packages/loader/container-loader/src/container.ts

```diff
@@ -1276,7 +1280,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         this.propagateConnectionState();
     }
 
-    private async getDocumentStorageService(): Promise<IDocumentStorageService> {
+    private async connectToStorage(): Promise<IDocumentStorageService> {
```

Comment:
I'd just remove / inline this method - there are only two callers I believe and code would become more readable